packages/libwebsockets/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://libwebsockets.org
 TERMUX_PKG_DESCRIPTION="Lightweight C websockets library"
 TERMUX_PKG_LICENSE="LGPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="4.4.1"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="4.5.0"
+TERMUX_PKG_REVISION=2
```

Comment:
The revision should be removed when the version is incremented.

```suggestion
```

Reply:
Yep, that's usually how I grab the new checksum(s) for a package when updating them.